packages/puppeteer-core/src/api/Browser.ts

```diff
@@ -399,6 +399,9 @@ export abstract class Browser extends EventEmitter<BrowserEvents> {
    *
    * {@link Page | Pages} can override the user agent with
    * {@link Page.setUserAgent}.
+   *
+   * @privateRemarks BiDi
+   * Not supported when using protocol `webDriverBiDi`
```

Comment:
We will need to distinguish between BiDi and BiDi over CDP since we can support more things there. Also, let's write the remarks in the following way, wdyt?

```
{@link gettingStartedWithProtocolsLink | CDP}: Supported
{@link gettingStartedWithProtocolsLink | WebDriver BiDi}: Supported/Not supported
```

Later we can add 

```
{@link gettingStartedWithProtocolsLink | WebDriver BiDi+}: Supported/Not supported
```

to the list